From akpm@osdl.org Thu May 18 01:13:22 2006
Message-Id: <200605180813.k4I8DG5e012968@shell0.pdx.osdl.net>
From: Eduard Warkentin <eduard.warkentin@gmx.de>
Subject: USB: added support for ASIX 88178 chipset USB Gigabit Ethernet adaptor
To: eduard.warkentin@gmx.de, greg@kroah.com, mm-commits@vger.kernel.org
Date: Thu, 18 May 2006 01:13:17 -0700

From: Eduard Warkentin <eduard.warkentin@gmx.de>

Add support for detection and dworking with a ASIX 88178 based USB-Gigabit
adaptor.  With the patch, it is detected and handled correctly by the asix
module.

Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/net/asix.c |    4 ++++
 1 file changed, 4 insertions(+)

--- gregkh-2.6.orig/drivers/usb/net/asix.c
+++ gregkh-2.6/drivers/usb/net/asix.c
@@ -912,6 +912,10 @@ static const struct usb_device_id	produc
         USB_DEVICE (0x0b95, 0x7720),
         .driver_info = (unsigned long) &ax88772_info,
 }, {
+	// ASIX AX88178 10/100/1000
+	USB_DEVICE (0x0b95, 0x1780),
+	.driver_info = (unsigned long) &ax88772_info,
+}, {
 	// Linksys USB200M Rev 2
 	USB_DEVICE (0x13b1, 0x0018),
 	.driver_info = (unsigned long) &ax88772_info,
